library/webpack-plugins/react-universal-plugin.js

```diff
@@ -114,6 +117,54 @@ module.exports = function reactUniversalPlugin () {
           done()
         }
       })
+
+      // make sure the __webpack_js_client_chunk_hashes__ is available in modules (code copied from ExtendedApiPlugin)
+      compiler.plugin('compilation', (compilation, { normalModuleFactory }) => {
```

Comment:
The pattern of adding a variable to the compilation is used in quite a few places, we should probably move it into a `lib` function.